Common/UnitDefinitions/RotationalStiffness.json

```diff
@@ -20,6 +20,30 @@
           "Abbreviations": [ "N·m/rad", "Nm/rad" ]
         }
       ]
+    },
+    {
+      "SingularName": "PoundForceFootPerDegrees",
+      "PluralName": "PoundForceFeetPerDegrees",
+      "FromUnitToBaseFunc": "x*77.6826",
+      "FromBaseToUnitFunc": "x/77.6826",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lbf·ft/deg" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilopoundForceFootPerDegrees",
+      "PluralName": "KilopoundForceFeetPerDegrees",
+      "FromUnitToBaseFunc": "x*77682.6",
+      "FromBaseToUnitFunc": "x/77682.6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kipf·ft/deg" ]
```

Comment:
Awaiting decision in #748 on kipf vs kip.

Reply:
Should we use degree symbol in the abbreviation? Below ref uses `kips.ft.s/°`.

Ref: http://webcache.googleusercontent.com/search?q=cache:BzUwiIqqit4J:52.202.177.230/units_reference_guide.html&hl=en&gl=no&strip=1&vwsrc=0

Reply:
Personally I think it's best to stick with "deg", to be consistent with "rad" if it's in radians.

But we could define all of the following?

1. kipf·ft/deg
2. kip·ft/deg
3. k·ft/deg
4. kipf·ft/°
5. kip·ft/°
6. k·ft/°

Reply:
I'm fine with deg, but I think we should add both as we have done for other quantities. We seem to have consistently used degree symbol as primary and "deg" as secondary.

https://github.com/angularsen/UnitsNet/search?l=JSON&q=deg

Reply:
So yes, I agree with your proposal if we choose this ordering:
1. Degree symbol, "deg"
2. kipf, kip, k (as discussed in #748 )